components/LinkIcon.tsx

```diff
@@ -0,0 +1,19 @@
+interface Props {
+  className?: string;
+}
+
+export const LinkIcon = ({ className }: Props) => {
```

Comment:
-1 here you're destructure the `className` prop but you're not passing it to the component. let's remove the `h-4 w-4 inline-block mr-2` and instead add `className={className}` and pass `h-4 w-4 inline-block mr-2` when the component is used